server/routes/aws/onboarding.py

```
@@ -398,3 +399,449 @@ def workspace_cleanup(workspace_id):
+                results.append({"accountId": account_id, "success": False, "error": f"accountId does not match role ARN (ARN has {arn_account})"})
+                continue
+
+            try:
+                assume_workspace_role(
+                    role_arn=role_arn,
+                    external_id=external_id,
+                    workspace_id=workspace_id,
+                    duration_seconds=900,
+                    region=region,
+                )
+            except Exception as assume_err:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)